{"name": "test_fail_password_empty[--对不起！请输入密码！]", "status": "passed", "attachments": [{"name": "stdout", "source": "595d9dc9-736f-41af-b934-cfb0408440d7-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "''"}, {"name": "password", "value": "''"}, {"name": "expect", "value": "'对不起！请输入密码！'"}], "start": 1601430092424, "stop": 1601430096255, "uuid": "0a9cad74-17aa-4a4b-88c6-4441eaf92bc1", "historyId": "19c20d6591c707086577ec0f634f58fd", "testCaseId": "e048db9676071fc597e1087551b47302", "fullName": "TestCases.test_1_loginCase.TestLogin#test_fail_password_empty", "labels": [{"name": "tag", "value": "Test1"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_1_loginCase"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "HPTSJ003"}, {"name": "thread", "value": "17988-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_1_loginCase"}]}